README.md

```diff
@@ -29,44 +30,93 @@ Note that `words` should be automatically installed by `pip` when you follow
 the installation instructions for this package.
 
 ## Directory Structure
+
 * `analysis_config/` contains a configuration file which can be used for analysis of cost, latency, and performance metrics for one set of hyperparameters for a particular study. This config is used in the script scripts/anaylsis_for_one_study.py
```

Comment:
The paths for the various config directories should be updated to reflect the new directory hierarchy.

Reply:
will address this in the revision